utils/process-stats-dir.py

```diff
@@ -440,7 +440,12 @@ def evaluate(args):
     ident = re.compile('(\w+)$')
     for (k, v) in merged.stats.items():
         if k.startswith("time.") or '.time.' in k:
-            continue
+            # In merge-timers mode, we flatten timer names so they can be
+            # used as idents; otherwise we omit them since they're messy.
```

Comment:
This seems unrelated, but sure. I can't really review it because I don't know what problem it's solving.

Reply:
This is just plumbing to make it possible to evaluate the ratio I use in the test.